fades/logger.py

```diff
@@ -20,11 +20,35 @@
 import logging.handlers
 import os.path
 
+from fades._version import __version__
+
 
 FMT_SIMPLE = "*** fades ***  %(asctime)s  %(levelname)-8s %(message)s"
 FMT_DETAILED = "*** fades ***  %(asctime)s  %(name)-18s %(levelname)-8s %(message)s"
 FMT_SYSLOG = "[%(process)d] %(name)-18s %(levelname)-8s %(message)s"
 
+SALUTATION = "Hi! This is fades {}, automatically managing your dependencies".format(__version__)
+
+
+class SalutingStreamHandler(logging.StreamHandler):
+    """A handler that salutes once before polluting user screen.
+
+    Note that the salutation is done in INFO level, to respect "verbose" modifiers.
+    """
+
+    def __init__(self, logger):
```

Comment:
Perhaps this new class should take the salutation message as an argument instead of using a module level variable?

Reply:
You'd be gaining a flexibility that you're not using, with the cost of more complexity. So not worth it...